### PR TITLE
add simple selector for #1320

### PR DESCRIPTION
**Pull request by zyradyl**
another small PR for #1320, it's nothing pretty but it does work :)

i also merged the individual page bar into the area between the arrows to keep things a bit neater.

before (taken from KopiaUI):

after:

new bar + selector:

options:

resized page:

example of what it looks like when there are more than 20 pages (had to tweak page size for a demo):

### CODE REVIEW

File: htmlui/src/Table.js [hunk 1]
```diff
@@ -59,11 +59,12 @@ export default function MyTable({ columns, data }) {
...
     nextPage,
     previousPage,
-    state: { pageIndex },
+    setPageSize,
+    state: { pageIndex, pageSize },
   } = useTable({
     columns,
     data,
-    initialState: { pageSize: 10 },
+    initialState: { pageSize: 3 },
```
Comment (jkowalski) [inline htmlui/src/Table.js:67]: 10 perhaps?

File: htmlui/src/Table.js [hunk 2]
```diff
@@ -80,13 +81,14 @@ export default function MyTable({ columns, data }) {
...
+      {paginationItems(pageOptions.length, pageIndex+1, gotoPage)}
       <Pagination.Next onClick={() => nextPage()} disabled={!canNextPage} />
       <Pagination.Last onClick={() => gotoPage(pageCount - 1)} disabled={!canNextPage} />
     </Pagination>
-
-    &nbsp;
     <Pagination size="sm" variant="dark">
-      {paginationItems(pageOptions.length, pageIndex+1, gotoPage)}
+    <select value={pageSize} onChange={e => {setPageSize(Number(e.target.value))}}>
+      {[10, 20, 30, 40, 50].map(pageSize => (<option key={pageSize} value={pageSize}>Show {pageSize}</option>))}
```
Comment (jkowalski) [inline htmlui/src/Table.js:90]: would it hurt to add 100 or something?

**Comment by jkowalski** [comment]
This is very cool, we can longer-term work on keeping this persistent across windows and repos, but it's a great start. I have just minor comments.

File: htmlui/src/Table.js (hunk 2 above)
Comment (zyradyl) [reply htmlui/src/Table.js:90]: not at all, i will do that now!

File: htmlui/src/Table.js (hunk 1 above)
Comment (zyradyl) [reply htmlui/src/Table.js:67]: oops, this was put to three just for that last screenshot and i forgot to change it back.

**Pull request merged**